pyproject.toml

```diff
@@ -1,3 +1,86 @@
+[tool.poetry]
+name = "fedora_messaging"
+version = "3.4.1"
+description = "A set of tools for using Fedora's messaging infrastructure"
+authors = [
+    "Fedora Infrastructure <admin@fedoraproject.org>"
+]
+license = "GPLv2+"
+readme = "README.rst"
+repository = "https://github.com/fedora-infra/fedora-messaging"
+maintainers = ["Fedora Infrastructure Team"]
+classifiers = [
+        "Development Status :: 5 - Production/Stable",
+        "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+    ]
+keywords = ["fedora"]
+packages = [
+    { include = "fedora_messaging" },
+]
+include = [
+    { path = "docs", format = "sdist" },
+		{ path = "CODEWNERS", format = "sdist"},
```

Comment:
There's an alignment issue here. Also, I think the `CODEOWNERS` file is only used by Github, no need to include it in the sdist.